src/main/java/org/spongepowered/common/data/key/KeyRegistry.java

```diff
@@ -61,6 +62,8 @@ public static void registerKeys() {
         keyMap.put("velocity", makeSingleKey(Vector3d.class, Value.class, of("Velocity")));
         keyMap.put("eye_height", makeSingleKey(Double.class, Value.class, of("EyeHeight")));
         keyMap.put("eye_location", makeSingleKey(Vector3d.class, Value.class, of("EyeLocation")));
+        keyMap.put("fire_ticks", makeSingleKey(Integer.class, Value.class, of("FireTicks")));
```

Comment:
These need to be MutableBoundedValue, not Value. We need to make sure that we are using the correct types.